src/main/java/bit/javaoop/Company.java

```diff
@@ -0,0 +1,46 @@
+package bit.javaoop;
+
+public class Company {
+    private String companyName;
+    private CEO ceo;
+
+
+    public void setCompanyName(String companyName) {
+        this.companyName = companyName;
+    }
+
+    public CEO hireCEO()
+    {
+        CEO ceo1 = CEO.getInstance();
+        ceo=ceo1;
+        ceo.setName("Krzychu"); //setting name e.g from keyboard
+        ceo.addEmployee(ceo);
+        return ceo;
+    }
+
+
+
+    public String toString() {
+        String hierarchy="";
+        for(AbstractEmployee e: ceo.getEmployeeList())
+        {
+            switch (e.getPosition())
+            {
+                case "ceo":
+                    hierarchy="\nCEO - " + e.getName()+hierarchy;
+                    break;
+                case "manager":
+                    hierarchy+=("\n\tManager - "+e.getName());
```

Comment:
copy-paste from another review:
--------

If you use IntelliJ it will tell you that:

<img width="465" alt="screen shot 2018-03-26 at 21 14 35" src="https://user-images.githubusercontent.com/17063095/37927548-bc3370fe-313a-11e8-8655-d1c874ee4de7.png">

It means that you should use a `StringBuilder` instead. 
Strings in Java are implemented using a Flyweight design pattern (https://en.wikipedia.org/wiki/Flyweight_pattern) which means that every time you append something to a String it creates a new objected which makes it very inefficient in terms of memory.